Swift/KVSiOSApp/Constants.swift

```diff
@@ -4,10 +4,10 @@ import AWSCognitoIdentityProvider
 // Cognito constants
 let awsCognitoUserPoolsSignInProviderKey = "UserPool"
 
-let cognitoIdentityUserPoolRegion = AWSRegionType.Unknown //  <- REPLACE ME!
+let cognitoIdentityUserPoolRegion = AWSRegionType.Unknown // <- REPLACE ME!
 let cognitoIdentityUserPoolId = "REPLACEME"
 let cognitoIdentityUserPoolAppClientId = "REPLACEME"
-let cognitoIdentityUserPoolAppClientSecret = "REPLACEME"
+let cognitoIdentityUserPoolAppClientSecret = "REPLACEME" // <- Leave blank "" if AppClient does not use a client secret.
```

Comment:
I believe you would also need to set this to `nil` instead?

https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios/blob/eeaef50bd5cc10a07bb5d8b5b1dc9c93ac800687/Swift/KVSiOSApp/AppDelegate.swift#L36

It makes logical sense considering the "credentialsProvider" argument in the previous line.